src/core/default_appearance.js

```diff
@@ -102,4 +116,488 @@ function createDefaultAppearance({ fontSize, fontName, fontColor }) {
   )}`;
 }
 
-export { createDefaultAppearance, getPdfColor, parseDefaultAppearance };
+function canUseFakeFont() {
+  return FeatureTest.isOffscreenCanvasSupported;
```

Comment:
This must take the newly added `isOffscreenCanvasSupported` API-option into account as well, please note how this is done in e.g. `src/core/image.js` (was introduced in PR #15542).